elasticjob-lite/elasticjob-lite-core/src/test/java/org/apache/shardingsphere/elasticjob/lite/internal/guarantee/GuaranteeServiceTest.java

```diff
@@ -143,4 +152,32 @@ public void assertClearAllCompletedInfo() {
         guaranteeService.clearAllCompletedInfo();
         verify(jobNodeStorage).removeJobNodeIfExisted("guarantee/completed");
     }
+
```

Comment:
Does it work without `@Test` annotation?

Reply:
> Does it work without `@Test` annotation?

done